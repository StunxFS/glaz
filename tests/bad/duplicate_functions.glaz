fn xx() {}

fn xx() {}

fn main() {
  xx();
}
