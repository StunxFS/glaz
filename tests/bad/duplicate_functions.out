tests/bad/duplicate_functions.glaz:3:5: error: the name `xx` is defined multiple times
     | 
   3 | fn xx() {}
     |    ^^
glaz error: could not compile `duplicate_functions`, aborting due to previous error.
